README.md

```diff
@@ -1 +1,94 @@
-# terra-framework
\ No newline at end of file
+<!-- Logo -->
+<p align="center">
+  <img height="128" width="128" src="https://github.com/cerner/terra-framework/raw/master/terra.png">
+</p>
+
+<!-- Name -->
+<h1 align="center">
+  Terra Framework
+</h1>
+
+[![Cerener OSS](https://img.shields.io/badge/Cerner-OSS-blue.svg?style=flat)](http://engineering.cerner.com/2014/01/cerner-and-open-source/)
+[![Build Status](https://travis-ci.org/cerner/terra-framework.svg?branch=master)](https://travis-ci.org/cerner/terra-framework)
+[![lerna](https://img.shields.io/badge/Maintained%20With-Lerna-cc00ff.svg)](https://lernajs.io/)
+[![devDependencies status](https://david-dm.org/cerner/terra-framework/dev-status.svg)](https://david-dm.org/cerner/terra-framework?type=dev)
+
+- [Supported Browsers](#supported-browsers)
+- [Packages](#packages)
+- [Internationalization](#internationalizationi18n)
```

Comment:
Do we need to include information regarding Internationalization if the packages being placed in this repo are to build application frameworks

Reply:
I can remove it and add it back if a package utilizing it is added. I can't think of anything immediately that would care.